work_queue/src/work_queue_worker.c

```diff
@@ -442,6 +451,30 @@ static void report_worker_ready( struct link *master )
 	send_master_message(master,"workqueue %d %s %s %s %d.%d.%d\n",WORK_QUEUE_PROTOCOL_VERSION,hostname,os_name,arch_name,CCTOOLS_VERSION_MAJOR,CCTOOLS_VERSION_MINOR,CCTOOLS_VERSION_MICRO);
 	send_master_message(master, "info worker-id %s\n", worker_id);
 	send_features(master);
+
```

Comment:
Break this out into a separate function `send_tlq_config` or the like, to keep an even high level structure of `report_worker_ready`